app/views/citizens/other_assets/show.html.erb

```diff
@@ -0,0 +1,42 @@
+
+<%= render partial: 'shared/forms/error_summary', locals: { model: @form } %>
+
+  <%= form_with(model: @form, url: citizens_other_assets_path, method: :patch, local: true) do |form| %>
+    <div class="govuk-!-padding-bottom-8"></div>
+    <div class="govuk-!-padding-bottom-8"></div>
+
+    <div class="govuk-form-group">
+      <fieldset class="govuk-fieldset" aria-describedby="citizenship-conditional-hint">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+          <h1 class="govuk-fieldset__heading">
+            <%= t('.h1-heading') %>
+          </h1>
+        </legend>
+
+        <div class="govuk-checkboxes" data-module="checkboxes">
+          <%= render partial: 'second_home_conditional_checkbox' %>
+          <%= render partial: 'shared/forms/conditional_checkbox_with_numeric_input',
+                     locals: { attr: :timeshare, label: t('.timeshare'), hint: t('.value_after_loan') } %>
```

Comment:
I just merged my PR so if you want you can reuse the partial I created if it makes sense (https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/master/app/views/citizens/savings_and_investments/_attribute.html.erb)
Feel free to move it to `/shared/` and rename it if you do use it